File: C# Fundamentals May 2019/04.Data Types and Variables/PHP/SpecialNumbers.php
<?php
$number = intval(readline());

for ($i = 1; $i <= $number; $i++) {
    $digits = $i;
    $sum = 0;
    while ($digits > 0) {
        $digit = $digits % 10;
        $sum += $digit;

        $digits/=10;
    }

    if ($sum === 5 || $sum === 7 || $sum === 11) {
        echo "$i -> True".PHP_EOL;
    } else {
        echo "$i -> False".PHP_EOL;
    }
}
?>

File: C# Fundamentals May 2019/04.Data Types and Variables/Java/src/SpecialNumbers.java
import java.util.Scanner;
public class SpecialNumbers {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int number = Integer.parseInt(scanner.nextLine());

        for (int i = 1; i <=number ; i++) {
            int currentNumber = i;

            int sum = 0;
            while (currentNumber > 0) {
                int digit = currentNumber % 10;
                sum += digit;
                currentNumber /=10;
            }

            if (sum == 5 || sum == 7 || sum == 11) {
                System.out.printf("%d -> True%n", i);
            } else {
                System.out.printf("%d -> False%n", i);
            }
        }
    }
}
